docs/products/images/visit_image.rst

```diff
@@ -65,6 +65,13 @@ The metadata for visit images retrieved from the Butler include
 information about the observation (e.g., pointing, weather),
 and the derived PSF, photometric calibration, and WCS.
 
+WCS
+"""
+
+The World Coordinate System objects for difference images are not exactly representable as FITS; the FITS headers have an approximation that is good enough only for visualization.
```

Comment:
Copy-paste typo? It says "difference images", but the file is for `visit_image`s.